src/components/DraggableList/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useCallback} from 'react';
+import {DragDropContext, Droppable, Draggable, type OnDragEndResponder, type OnDragUpdateResponder} from 'react-beautiful-dnd';
+import {ScrollView} from 'react-native';
+import useDraggableInPortal from './useDraggableInPortal';
+import type {DraggableListProps} from './types';
+import styles from '../../styles/styles';
+
+type ReorderParams<T> = {
+    list: T[];
+    startIndex: number;
+    endIndex: number;
+};
+
+// Function to help us with reordering the result
```

Comment:
Could you provide more explanation? Does this function move the element from `startIndex` to `endIndex`?

Reply:
nice! thank you for doing that